pkg/streamingpromql/engine_test.go

```diff
@@ -39,23 +38,21 @@ import (
 	"github.com/grafana/mimir/pkg/streamingpromql/testutils"
 	"github.com/grafana/mimir/pkg/streamingpromql/types"
 	"github.com/grafana/mimir/pkg/util/globalerror"
+	"github.com/grafana/mimir/pkg/util/promqlext"
 )
 
 func init() {
 	types.EnableManglingReturnedSlices = true
+	// To test double_exponential_smoothing and holt_winters
+	promqlext.ExtendPromQL()
```

Comment:
This could go in an `init` function for the whole `streamingpromql` package, not just the tests.

Reply:
Updated in e7514ed403cf7987e3c4b42669317cf340430223 and dc1a3f2af27d5927646c08ff2b6c55c44d49a3b7